src/jit/emitxarch.cpp

```diff
@@ -75,27 +75,125 @@ bool emitter::IsAVXInstruction(instruction ins)
 // to indicate whether a 3-operand instruction.
 bool emitter::IsDstDstSrcAVXInstruction(instruction ins)
 {
-    return IsAVXInstruction(ins) &&
```

Comment:
👍 

I find the new code significantly easier to read and it should result in smaller diffs when/if the code changes in the future

Reply:
Thnx 
Actually it started to be quite difficult to go through all instructions. Now they are sorted and in single column should be both faster and easier to read 😄

Reply:
Fixed

Reply:
Could you pull this out into a separate PR? It would help minimize conflicts between PRs if we could get it in first (assuming it gets approved of course).

Reply:
@tannergooding  Will do it tomorrow